### DISCUSSION TITLE
Add command to install all packages

### DISCUSSION THREAD

**Discussion by dingedi** [argosopentech/argos-translate#177] [Ideas]
I have been using argostranslate for about 2 weeks and love it! Thank you very much for your work.

I had to install it several times, and I wanted to install all the packages each time and I had to do it one by one. 
I did it manually the first time ( it was a bit boring 😅 ) but i quickly added an `argospm install-all` command that installs all the packages

Are you interested in a PR?

**Comment by PJ-Finlay** [comment]
Yes! That's probably useful. The plan is to do other things like data with Argospm so let's do a `translate` "metapackage" that installs all the translations. 

`argospm install translate`

**Reply by dingedi** [reply]
Done here [https://github.com/argosopentech/argos-translate/pull/179](https://github.com/argosopentech/argos-translate/pull/179)

**Comment by lovetocode14** [comment]
When running the argospm install translate command from venv in PyCharm I am getting Package not found as output.

**Reply by PJ-Finlay** [reply]
This hadn't been pushed to PyPI, just pushed 1.5.2.

**Reply by lovetocode14** [reply]
Ah k
